### PR TITLE
Add managed host for functioning webhook callbacks

### PR DESCRIPTION
**Pull request by begizi**

### CODE REVIEW

**Comment by aaddaamm** [comment]
👍

File: api/src/helios/github/github.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func Service() helios.ServiceHandler {
 
 		// Setup Goth Authentication
 		goth.UseProviders(
-			githubProvider.New(g.githubKey, g.githubSecret, fmt.Sprintf("http://localhost:%s/auth/github/callback", h.Config.GetString("port")), "repo", "user:email"),
+			githubProvider.New(g.githubKey, g.githubSecret, fmt.Sprintf("http://$s:%s/auth/github/callback", h.Config.GetString("host"), h.Config.GetString("port")), "repo", "user:email"),
```
Comment (sethkrasnianski) [inline api/src/helios/github/github.go]: Isn't that supposed to be `%s`, not `$s`?

Comment (begizi) [inline api/src/helios/github/github.go]: Yes. Good catch

Comment (begizi) [inline api/src/helios/github/github.go]: Updated

**Pull request merged**